qrenderdoc/Windows/PerformanceCounterViewer.cpp

```diff
@@ -223,9 +205,32 @@ void PerformanceCounterViewer::CaptureCounters()
 
       QStringList headers;
       headers << lit("EID");
+      TimeUnit timeunit = m_Ctx.Config().EventBrowser_TimeUnit;
       for(const CounterDescription &cd : counterDescriptions)
       {
-        headers << cd.name;
+        QString unit = QString::null;
+        switch(cd.unit)
+        {
+          case CounterUnit::Bytes: unit = lit("bytes"); break;
+
+          case CounterUnit::Cycles: unit = lit("cycles"); break;
+
+          case CounterUnit::Percentage: unit = lit("%"); break;
+
+          case CounterUnit::Seconds: unit = UnitSuffix(timeunit); break;
+
+          case CounterUnit::Absolute:
+          case CounterUnit::Ratio: break;
+
+          case CounterUnit::Hertz: unit = lit("Hz"); break;
+          case CounterUnit::Volt: unit = lit("V"); break;
+          case CounterUnit::Celsius: unit = lit("°C"); break;
+        }
+
+        if(unit.isNull())
+          headers << cd.name;
+        else
+          headers << lit("%1 [%2]").arg(cd.name, unit);
```

Comment:
Can you use parenthesis like `"%1 (%2)"`? That feels more natural to me.

Reply:
done